srcDLL/DllMain.cpp

```diff
@@ -74,9 +73,7 @@ int TApp::Init()
 	bool success = Op2UnprotectMemory(destinationBaseAddress, 0x00587000 - 0x00585000);
 
 	if (!success) {
-		std::ostringstream stringStream;
-		stringStream << "Error unprotecting memory at: 0x" << std::hex << destinationBaseAddress << ".";
-		PostError(stringStream.str());
+		PostError("Error unprotecting memory at: 0x" + AddrToHexString(destinationBaseAddress));
```

Comment:
Happy to see this error message formatting simplified by moving the formatting code out of OP2Memory.